Description: Introduce --with-app-profile to change the profile location, so
 that we can have nightly builds that are fully coinstallable with release
 builds
Author: Chris Coulson <chris.coulson@canonical.com>
Forwarded: no

Index: firefox-trunk-51.0~a1~hg20160830r311818/old-configure.in
===================================================================
--- firefox-trunk-51.0~a1~hg20160830r311818.orig/old-configure.in
+++ firefox-trunk-51.0~a1~hg20160830r311818/old-configure.in
@@ -2448,6 +2448,15 @@ if test -n "$WITH_APP_BASENAME" ; then
     MOZ_APP_BASENAME="$WITH_APP_BASENAME"
 fi
 
+MOZ_ARG_WITH_STRING(app-profile,
+[--with-app-profile=APPPROFILE sets MOZ_APP_PROFILE to APPPROFILE],
+WITH_APP_PROFILE=$withval,
+)
+
+if test -n "$WITH_APP_PROFILE" ; then
+    MOZ_APP_PROFILE="$WITH_APP_PROFILE"
+fi
+
 # Special cases where we need to AC_DEFINE something. Also a holdover for apps
 # that haven't made a confvars.sh yet. Don't add new stuff here, use
 # MOZ_BUILD_APP.
Index: firefox-trunk-51.0~a1~hg20160830r311818/build/moz.configure/old.configure
===================================================================
--- firefox-trunk-51.0~a1~hg20160830r311818.orig/build/moz.configure/old.configure
+++ firefox-trunk-51.0~a1~hg20160830r311818/build/moz.configure/old.configure
@@ -255,6 +255,7 @@ def old_configure_options(*options):
     '--with-android-sdk',
     '--with-app-basename',
     '--with-app-name',
+    '--with-app-profile',
     '--with-arch',
     '--with-branding',
     '--with-crashreporter-enable-percent',
